src/components/BookList.js

```diff
@@ -1,57 +1,65 @@
-              </p>
-              <div className="comment">
-                <button type="submit" className="i" id="btn">comments</button>
-                <div className="border" />
-                <button type="submit" className="i" id="btn">Remove</button>
-                <div className="border" />
-                <button type="submit" className="i" id="btn">Edit</button>
+function BookList({ books }) {
+  const dispatch = useDispatch();
+  const handleDelete = (itemId) => {
+    dispatch(removeBook(itemId));
+  };
+  return (
+    <div>
+      <ul className="card">
+        {books.map((book) => (
+          <li key={book.item_id} className="cardlist">
+            <div className="columns">
+              <div className="col1">
+                <p>{book.category}</p>
+                <h1>{book.title}</h1>
+                <p id="author" className="i">
+                  {book.author}
+                </p>
+                <div className="comment">
+                  <button type="submit" className="i" id="btn">comments</button>
+                  <div className="border" />
+                  <button type="submit" className="i" id="btn" onClick={() => handleDelete(book.item_id)}>Remove</button>
+                  <div className="border" />
+                  <button type="submit" className="i" id="btn">Edit</button>
+                </div>
               </div>
-            </div>
 
-            <div className="col2">
-              <div className="progress" />
-              <div className="percentage">
-                <p className="Percent">
-                  {book.percentageComplete}
-                  %
-                </p>
-                <p>Completed</p>
+              <div className="col2">
+                <div className="progress" />
+                <div className="percentage">
+                  <p className="Percent">
+                    {book.percentageComplete}
+                    %
+                  </p>
+                  <p>Completed</p>
+                </div>
               </div>
-            </div>
 
-            <div className="border" />
+              <div className="border" />
 
-            <div className="col3">
-              <p>CURRENT CHAPTER</p>
-              <p>{book.currentChapter}</p>
-              <button type="button">UPDATE PROGRESS</button>
+              <div className="col3">
+                <p>CURRENT CHAPTER</p>
+                <p>{book.currentChapter}</p>
+                <button type="button">UPDATE PROGRESS</button>
+              </div>
             </div>
-          </div>
-        </li>
-      ))}
-    </ul>
-  </div>
-);
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+}
 
 BookList.propTypes = {
   books: PropTypes.arrayOf(
     PropTypes.shape({
-      id: PropTypes.number.isRequired,
+      item_id: PropTypes.number.isRequired,
       category: PropTypes.string.isRequired,
       name: PropTypes.string.isRequired,
       author: PropTypes.string.isRequired,
```

Comment:
-  You have done great so far :+1:  but there are a few changes you need to make before moving to the next milestone. :smile: :wave: 

- [ ] Looking at the screenshot below, you can see that there is an error in the console with your `item_id` prop validation. I'll suggest you fix as it might affect the correct functionality of the app.

- - Screenshot:
 
![Selection_014](https://github.com/fatmahussein/book-store/assets/107632780/353e850a-44cd-4105-8f11-6f1a2627bbe3)